src/main/app/src/app/taken/taak-view/formio/formio-setup-service.ts

```diff
@@ -235,4 +242,44 @@ export class FormioSetupService {
         ),
     };
   }
+
+  private initializeAvailableDocumentsFieldsetComponent(
+    fieldsetComponent: ExtendedComponentSchema,
+  ): void {
+    if (!this.taak) {
+      return;
+    }
+
+    const documentViewComponent = fieldsetComponent.components?.find(
+      (component: { key: string }) => component.key === "ZAC_Documents",
```

Comment:
I would go with component type here. Relying on key is not ok, as form developers might have multiple controls with documents, hence different keys would be needed.